source/DSCResources/DSC_CMSoftwareUpdatePointComponent/DSC_CMSoftwareUpdatePointComponent.psm1

```diff
@@ -0,0 +1,1517 @@
+                Exclude      = $ProductsToExclude
+                CurrentState = $state.Products
+            }
+
+            $productsCompare = Compare-MultipleCompares @productsArray
+
+            if ($PSBoundParameters.ContainsKey('Products'))
+            {
+                if ($PSBoundParameters.ContainsKey('ProductsToInclude') -or $PSBoundParameters.ContainsKey('ProductsToExclude'))
+                {
+                    Write-Warning -Message $script:localizedData.ProductsIgnore
+                }
+            }
+
+            if ($productsCompare.Missing)
+            {
+                Write-Verbose -Message ($script:localizedData.ProductsMissing -f ($productsCompare.Missing | Out-String))
+                $result = $false
+            }
+
+            if ($productsCompare.Remove)
+            {
+                Write-Verbose -Message ($script:localizedData.ProductsRemove -f ($productsCompare.Remove | Out-String))
+                $result = $false
+            }
+        }
+
+        if ($UpdateClassifications -or $UpdateClassificationsToInclude -or $UpdateClassificationsToExclude)
+        {
+            if ($UpdateClassificationsToInclude -and $UpdateClassificationsToExclude)
+            {
+                foreach ($item in $UpdateClassificationsToInclude)
+                {
+                    if ($UpdateClassificationsToExclude -contains $item)
+                    {
+                        Write-Warning -Message ($script:localizedData.UpdateClassInEx -f $item)
+                        $result = $false
+                    }
+                }
+            }
+
+            $updateClassificationsArray = @{
+                Match        = $UpdateClassifications
+                Include      = $UpdateClassificationsToInclude
+                Exclude      = $UpdateClassificationsToExclude
+                CurrentState = $state.UpdateClassifications
+            }
+
+            $updateClassificationsCompare = Compare-MultipleCompares @updateClassificationsArray
+
+            if ($PSBoundParameters.ContainsKey('UpdateClassifications'))
+            {
+                if ($PSBoundParameters.ContainsKey('UpdateClassificationsToInclude') -or $PSBoundParameters.ContainsKey('UpdateClassificationsToExclude'))
+                {
+                    Write-Warning -Message $script:localizedData.UpdateClassificationsIgnore
+                }
+            }
+
+            if ($updateClassificationsCompare.Missing)
+            {
+                Write-Verbose -Message ($script:localizedData.UpdateClassificationsMissing -f ($updateClassificationsCompare.Missing | Out-String))
+                $result = $false
+            }
+
+            if ($updateClassificationsCompare.Remove)
+            {
+                Write-Verbose -Message ($script:localizedData.UpdateClassificationsRemove -f ($updateClassificationsCompare.Remove | Out-String))
+                $result = $false
+            }
+        }
+
+        if ($ScheduleType)
```

Comment:
maybe consider a check if they have additional schedule settings but don't have scheduletype a warn in test and throw in set?

Reply:
fixed.